.github/workflows/ci.yml

```diff
@@ -186,6 +186,9 @@ jobs:
       NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
       NX_BRANCH: ${{ github.event.number || github.ref_name }}
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    permissions:
```

Comment:
@jackw I've added this because I assume it will be required. fyi

Reply:
Looking at [the docs](https://docs.npmjs.com/generating-provenance-statements#example-github-actions-workflow) we do need to set the `id-token`. However are we using this github token or the `github-app-token` further down?

Reply:
My assumption is that they are a bit different and it needs the specific id-token. I guess we will see once we run it :)